externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/spark/kyuubi/SparkSQLEngineDeregisterSuite.scala

```diff
@@ -72,8 +72,10 @@ class SparkSQLEngineDeregisterMsgSuite extends SparkSQLEngineDeregisterSuite {
   override def withKyuubiConf: Map[String, String] = {
     super.withKyuubiConf ++ Map(ENGINE_DEREGISTER_EXCEPTION_MESSAGES.key ->
       // see https://issues.apache.org/jira/browse/SPARK-38926
-      // upper case SQL types in error messages
-      "to int causes overflow,to INT causes overflow")
+      // Upper case SQL types in error messages
+      // see https://issues.apache.org/jira/browse/SPARK-39007
+      // Use double quotes for SQL configs in error messages
+      "to int causes overflow,to \"INT\" causes overflow")
```

Comment:
does it work across spark versions? shall we use `to int causes overflow`?

Reply:
The current change is to support Spark 3.3 and 3.4 and previous releases, or we just use `causes overflow` to match?